doc/flux_to_redux.md

```diff
@@ -0,0 +1,134 @@
+
+The Influx architecture is not a required part of the process, and may make your life more difficult by adding extra steps. There are both advantaces and disadvantages, and assessing how large complexity of the refactor will likely provide the largest indication of whether or not it makes sense.
+
+### Advantages
+The main advantages are that
+- You can split the effort into very tangible subtasks. The first is refactoring the state archiecture (with a light component refactor), then each component can be its own task done over time. This is more notable in libaries that have components that will be challenging to update to `Component2`. Since `connected` components need to be `Component2`, the Redux refactor may be blocked by those efforts. This option allows to update the state to a Redux friendly architecture, and then as you update components you can update them to `Component2` and `connect` at the same time.
+- The workflow is much less complex. While heavily piggy backed off the first advantage, it's worth noting the workflow benefits of the incremental update. There shouldn't be as many massive merges or code reviews, and the granularity of tasks should make it more clear where manual testing is needed and when tests need to be written / updated. Ultimately this reduces the project complexity and the risk of regressions.
+- The actual update process can be easier to reason about in complex scenarios. Influx merges Flux and Redux to provide a "halfway" point that is a fairly straightforward transition both from Flux and to Redux. For those complex scenarios, it may seem daunting to transition straight to Redux, and Influx can lower the barrier. 
+
+### Disadvantages
+The biggest disadvantages are that it
+- Takes extra time as there are ultimately two refactors instead of one (however minimal the second one will be). While ideally the second refactor (from Influx to Redux) should be an easy lift, it's still ultimately unnecessary.
+- Does not provide any performance gains. The Redux `connected` components will update on every store update as long as Flux is in the mix. This mimics the behavior of Flux, but takes away the largest benefit of connecting to Redux.
+- Can add increased complexity and code to a possibly already complex state architecture. There is additional boilerplate and utilities necessary to maintain an Influx architecture that are completely unnecessary to Redux, making Influx more verbose and a little more confusing.
+
+In summary, it never makes sense to add steps and refactoring if you can afford not to. The primary advantage in Influx is that you can break up the project. If that may become important or is enticing, it is an option!
+
+## Example Explained
+This section goes over the example in `/web/flux_to_redux/advanced/` and includes the goal of each phase and the steps to go from the last phase (starting with Flux) to the current phase.
```

Comment:
I think it'd be helpful to also clarify that there's also a simpler example and link to it for users that have simpler use cases, so they don't have to read through all of this when they don't need to.